utils/led.c

```diff
@@ -27,17 +27,21 @@ volatile uint32_t g_gpio_pattern = LEDS_MASK;
 static uint8_t num_of_blinks, count;
 static uint32_t timer2_load_value = MSS_SYS_APB_0_CLK_FREQ;
 
+/* Local function prototypes */
+void led_set_high(void);
+void led_conf_tim2_periodic(void);
```

Comment:
I do not see this function as being used outside the LED module. Therefore, I would:

1. Make it `static`
2. Rename it to `tim2_start()`, or `start_led_timer()`

Reply:
Yes boss! You are right, I added `static` keyword to both local fns and renamed timer function to `led_tim2_start()`. I want `led_` prefix to be there to indicate its origin module

Reply:
OK, if you like it that way, it's fine with me. I don't see a point in knowing what the originating module is if the function is static and it cannot be used outside the module, but if you understand it better like this, fine.